src/main/java/org/kohsuke/github/GHGist.java

```diff
@@ -139,31 +153,7 @@ public GHGistFile getFile(String name) {
      * @return the files
      */
     public Map<String, GHGistFile> getFiles() {
-        return Collections.unmodifiableMap(files);
-    }
-
-    GHGist wrapUp(GHUser owner) {
-        this.owner = owner;
-        this.root = owner.root;
-        wrapUp();
-        return this;
-    }
-
-    /**
-     * Used when caller obtains {@link GHGist} without knowing its owner. A partially constructed owner object is
-     * interned.
-     */
-    GHGist wrapUp(GitHub root) {
-        this.owner = root.getUser(owner);
-        this.root = root;
-        wrapUp();
-        return this;
-    }
-
-    private void wrapUp() {
-        for (Entry<String, GHGistFile> e : files.entrySet()) {
-            e.getValue().fileName = e.getKey();
-        }
```

Comment:
Actions that were being done after instantiation are now done in the constructor, allowing for those values to be `final`.